### DISCUSSION TITLE
Compativel SC-10 (28461) ?

### DISCUSSION THREAD

**Discussion by djgel** [Yogui79/IntexPureSpa#10] [Q&A] [answered]
Hi, 

I have a  SC-10 (BR) , this model e manual is same a  SC-20 .... Find model, this is 28461 ... sometimes the same code is even referenced to the sc-20 version ... you know if this work for me ?

**Comment by Yogui79** [comment]
Hello

Honestly we haven't tested for this model but the remote look like mine
Could you post a picture
Do you have on the bottom right from the screen a blue led
Is your spa a 110V version?
@LEECHER1  please could you confirm

**Reply by djgel** [reply]
Yes, have a  blue led blink all time!
Yes, 110v Version!  1300W heating.

Well, I bought an LC12s on Aliexpress and I'm waiting for it to arrive next week and then test it with my ESP32, I've uploaded the codes on it...

The Manual is same models ... no find any diffences ...

**Comment by LEECHER1** [comment, marked as answer]
Hey, 

Yes it is the same version but with the 110v. 
We haven't no way to test it. 
Can you post it when it worked? Then we can include the article number in the code.

See this
[Picture](https://photos.app.goo.gl/CXTcm2g6j8tRLgwSA) 

Best regards

**Reply by djgel** [reply]
Hummmm Great information! BR is my country ... and SC-10 - 110v em SC-20 220v  ... manual only have this differences ... Tks @LEECHER1 Leecher!! tks @Yogui79 

I will test and return results for insert in repository :)

![20210617_114213](https://user-images.githubusercontent.com/1890658/122424154-df99a600-cf64-11eb-8382-e22b58074862.jpg)
![20210617_114202](https://user-images.githubusercontent.com/1890658/122424172-e2949680-cf64-11eb-9a41-19c23ca60c33.jpg)

Pictures for a  SC-10BR

**Reply by djgel** [reply]
Not good news, not find a  channel for SC-10 ... I think not using a 2.4ghz :(

**Reply by Yogui79** [reply]
hy @djgel 

strange are your sure your wiring is ok 

according the intex documentation it should be the same @LEECHER1 could you confirm

**Reply by LEECHER1** [reply]
Hey,
Yes the SC-10 is the same. 

Please check all things (cables, LC12s, the code, the distance to the Pumpe and LC12s 0-5m, read again the documentation.) Thanks.

**Reply by djgel** [reply]
Hi! This photos from my boards and connections
![20210719_113545](https://user-images.githubusercontent.com/1890658/126179691-66505bf7-b649-4006-9b33-bf05a2c2d434.jpg)
![20210719_113608](https://user-images.githubusercontent.com/1890658/126179698-e74a5ff5-89e4-4ac4-a1e2-2318e164fee6.jpg)
This is my connection in LC12s:
![Screenshot_20210719-114928_Gallery](https://user-images.githubusercontent.com/1890658/126179703-ee7aba3b-834e-485b-870e-51b77ac96b56.jpg)
![20210719_114240](https://user-images.githubusercontent.com/1890658/126179706-8aee73cb-2682-41fe-a73d-238a52cbd42d.jpg)

I tested all connection with multimeter.

Change  code and set more time to test a channel .... em stay a 1meter from pump ... 

if not stay quite in laptop, sometime led blink , not same a  demo vídeo.


possible LC12s  broken ?

**Reply by djgel** [reply]
https://youtu.be/Y2RKNXvps84  video scanning channel

**Reply by Yogui79** [reply]
Hi
Please could you try this
https://github.com/Yogui79/IntexPureSpa/issues/17
Look the last post

**Reply by djgel** [reply]
Hi,

Thanks for the feedback, sorry, my reading in English is great, but my writing is confusing! My problem is not in Search Network , it is in SearchChannel , the modification he made to search the channel, which is the time with the channel is open, I had also already changed it to a time of 5000 and nothing to find the channel. For this reason I considered that either my pump does not work on this frequency, or the lc12s is not operational. Worse, I ordered it to come from China because here in Brazil I couldn't find it to buy. New is another 1 month and not knowing where the problem is.

**Reply by djgel** [reply]
After many, many tests here, on channels that seemed to blink, absolute darkness I found the channel! The light blinks very weak, with this, I couldn't see! Serves for that other person who also did not find the channel, VERY hard to see. I had to go testing one by one to see this! another thing, the control has to be in the base to keep it on and work. I found it on Channel 1A! After this step, I went to the NettworkID search, and I found it in 30 Minutes, 0x103 ... Lucky! I moved to the main code, changing the network, MQTT, channel and network settings. However, I have an error after booting spa.ino:

20:53:42.982 -> 
20:53:42.982 -> ELF file SHA256: 000000000000000
20:53:42.982 -> 
20:53:42.982 -> Backtrace:0x40151ba0x3fb7900x4013f0b0x3fb7b 0x4008bc:0x3fb7d 0x40089ae0x3fb7f
20:53:42.982 -> 
20:53:42.982 -> Rebooting...
20:53:56.165 -> ets Jun  8 2016 00:22:57
20:53:56.165 -> 
20:53:56.165 -> rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
20:53:56.165 -> configsip: 0, SPIWP:0xee
20:53:56.212 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
20:53:56.212 -> mode:DIO, clock div:1
20:53:56.212 -> load:0x3fff0018,len:4
20:53:56.212 -> load:0x3fff001c,len:1044
20:53:56.212 -> load:0x40078000,len:10124
20:53:56.212 -> load:0x40080400,len:5856
20:53:56.212 -> entry 0x400806a8
20:53:56.492 -> ----------------------------------------------------------------------------------------------------------------------
20:53:56.492 -> 
20:53:56.492 ->           Intex Purespa communication sketch version 1.1.0 build on : 2021.05.29 22:25:18
20:53:56.492 -> 
20:53:56.492 -> ----------------------------------------------------------------------------------------------------------------------
20:53:56.539 -> Booting
20:53:57.520 -> Guru Meditation Error: Core  panic'ed (Interrupt wdt timeout on CPU0)
20:53:57.566 -> Core 0 register dump:
20:53:57.566 -> PC       0x40086ce PS       0x00060434  A0       0x8010a2a A1       0x3fb940  
20:53:57.566 -> A2       0x00000017  A3       0x00000000  A4       0x3fbf8  A5       0x00000040  
20:53:57.566 -> A6       0x00000000  A7       0x3fb200  A8       0x80083d7  A9       0x3fb930  
20:53:57.566 -> A10      0x00000000  A11      0x3fba0  A12      0x00000001  A13      0x00000001  
20:53:57.566 -> A14      0x00060420  A15      0x00000000  SAR      0x0000001e EXCCAUSE 0x00000005  
20:53:57.566 -> EXCVADDR 0x00000000  LBEG     0x4000ce  LEND     0x4000cf  LCOUNT   0xffff  
20:53:57.566 -> 
20:53:57.566 -> ELF file SHA256: 000000000000000
20:53:57.613 -> 
20:53:57.613 -> Backtrace:0x40086ce0x3fb9400x4010a27:0x3fb970 0x40089ae0x3fb9a
20:53:57.613 -> 
20:53:57.613 -> ore 1 register dump:
20:53:57.613 -> PC       0x40151ba PS       0x00060e4  A0       0x8013f0e A1       0x3fb790  
20:53:57.613 -> A2       0x00000000  A3       0x80000001  A4       0x00000000  A5       0x00000001  
20:53:57.613 -> A6       0x00060a0  A7       0x00000000  A8       0x00060023  A9       0x3fb88c 
20:53:57.613 -> A10      0x00000001  A11      0x00000001  A12      0x80089c0  A13      0x3fb690  
20:53:57.613 -> A14      0x00000000  A15      0x3fb480  SAR      0x00000000  EXCCAUSE 0x00000005  
20:53:57.613 -> EXCVADDR 0x00000000  LBEG     0x00000000  LEND     0x00000000  LCOUNT   0x00000000  
20:53:57.660 -> 
20:53:57.660 -> ELF file SHA256: 000000000000000
20:53:57.660 -> 
20:53:57.660 -> Backtrace:0x40151ba0x3fb7900x4013f0b0x3fb7b 0x4008bc:0x3fb7d 0x40089ae0x3fb7f
20:53:57.660 -> 
20:53:57.660 -> Rebooting...

Using ESP32 Dev Kit V1

**Reply by djgel** [reply]
finally , work here ! Received a V4, (V1  broken after tried 1000 times) and work , but  wifi reconnecting all time ... Downgrade a  Board EPS32 and work!    Happiness!
![image](https://user-images.githubusercontent.com/1890658/126870815-fe0fc30c-6c33-4b2d-9ea8-9cc3c68d8c85.png)

**Reply by LEECHER1** [reply]
Nice to hear to worked out :) 👍
